django/db/models.py

```diff
@@ -565,6 +565,26 @@ class Antibody(Reagent):
     labeling_details        = _TEXT(**_NULLOKSTR) 
     relevant_citations      = _TEXT(**_NULLOKSTR) 
     
+    @property
```

Comment:
It's unclear whether you intended to use the default None return value when these if's fall through, or you just forgot to write the else case. I am pretty sure you intended them to fall through and return None, so I would like to see an explicit `else` with `return None` on each of these. The other acceptable option is a ternary if/else expression, i.e. `return self.target_protein.name if self.target_protein else None`.